Makefile

```diff
@@ -145,11 +145,11 @@ endif
 endif
 endif
 
-module: $(BIN_OUTPUT_PATH)/viamrtsp
-	tar czf $(BIN_OUTPUT_PATH)/module.tar.gz $(BIN_OUTPUT_PATH)/viamrtsp
+module.tar.gz: $(BIN_OUTPUT_PATH)/viamrtsp
+	tar czf module.tar.gz -C $(BIN_OUTPUT_PATH) .
```

Comment:
now that I've gone though the process of uploading a module I do have some more feedback here:
Can we please produce tarballs under the architecture that they are for:
```
dist/linux/amd64/module.tar.gz
dist/linux/arm64/module.tar.gz
dist/android/arm64/module.tar.gz
dist/darwin/arm64/module.tar.gz
```

We can also change the target back to just `module`

Reply:
otherwise one could easily upload a module for one architecture to the registry under a different architecture. Splitting out the tarballs under a subdirectory for each platform reduces this risk.